ts/a11y/explorer/KeyExplorer.ts

```diff
@@ -1311,10 +1346,16 @@ export class SpeechExplorer
   protected nextSibling(node: HTMLElement): HTMLElement {
     const id = this.parentId(node);
     if (!id) return null;
-    const owns = this.getNode(id).getAttribute('data-semantic-owns')?.split(/ /);
+    const owns = this.getNode(id)
+      .getAttribute('data-semantic-owns')
+      ?.split(/ /);
     if (!owns) return null;
-    const i = owns.indexOf(this.nodeId(node));
-    return this.getNode(owns[i + 1]);
+    let i = owns.indexOf(this.nodeId(node));
+    let next = this.getNode(owns[++i]);
+    while (next && !next.hasAttribute('data-speech-node')) {
+      next = this.getNode(owns[++i]);
+    }
```

Comment:
Could this be

``` js
let next;
do {
  next = this.getNode(owns[++i]);
} while (next && !next.hasAttribute('data-speech-node'));
```

to avoid some redundancy?

Reply:
Done.